.svnignore

```diff
@@ -42,3 +42,4 @@ docker
 bin/pre-commit-hook.js
 bin/travis_install.sh
 yarn-error.log
+extensions/**/*.{js,jsx,json,gif,png,css,jpg,jpeg,sass,scss}
```

Comment:
Copying over https://github.com/Automattic/jetpack/pull/11639#discussion_r267776127 for visibility:

> To help future groking, I'd love for svnignore comments (I believe # preceding the line should work here too) to explain why these files are ignored. Someone new to JP development may be confused why we're ignoring jpg, css, etc.

Reply:
suggested by @dereksmart, could we do everything except `php` instead?

Reply:
See https://github.com/Automattic/jetpack/pull/11639#discussion_r267761123

Reply:
`svnignore` _is not a real thing_. Under the hood, this is relying on shell globbing:

https://github.com/Automattic/jetpack/blob/f8078c2cd12ac508334da2fb08e37a92cf283c14/tools/build-jetpack.sh#L70-L75

We should be extremely careful and keep that in mind.